client/src/components/Cards/Card.scss

```diff
@@ -62,69 +62,8 @@
     bottom: -20px;
   }
 
-  .display_more_btn {
```

Comment:
I delete this code because it was on `Card.scss` while changes where made on `CardList.scss`
So I prefer to keep it on `CardList.scss` (where butons are) to make reading easier